diff --git a/node_modules/npm-groovy-lint/lib/codenarc-caller.js b/node_modules/npm-groovy-lint/lib/codenarc-caller.js
index 7666df20..a27d4646 100644
--- a/node_modules/npm-groovy-lint/lib/codenarc-caller.js
+++ b/node_modules/npm-groovy-lint/lib/codenarc-caller.js
@@ -35,13 +35,13 @@ export class CodeNarcCaller {
     javaCallerOptions = {
         codeNarcServer: {
             minimumJavaVersion: 17,
-            maximumJavaVersion: 17,
+            maximumJavaVersion: 21,
             rootPath: __dirname,
             jar: "java/CodeNarcServer.jar",
         },
         codeNarcJava: {
             minimumJavaVersion: 17,
-            maximumJavaVersion: 17,
+            maximumJavaVersion: 21,
             rootPath: __dirname,
             jar: "java/CodeNarcServer.jar",
         },
